src/app/utils/utils.js

```diff
@@ -403,3 +405,7 @@ export const generateRelationIconImage = (applicationName, modelStatusData) => {
   }
   return generateIconImg(applicationName, application.charm);
 };
+
+export const formatFriendlyDateToNow = (date) => {
+  return formatDistanceToNow(parseISO(date), { addSuffix: true });
```

Comment:
Should we have this support a tooltip which has the actual value in it?

Reply:
There's already an issue to do that [here](https://github.com/canonical-web-and-design/jaas-dashboard/issues/965).. that why I made this a common util.